tf2_ros_py/tf2_ros/buffer_client.py

```diff
@@ -65,7 +66,7 @@ class BufferClient(tf2_ros.BufferInterface):
     def __init__(
         self,
         node: Node,
-        ns: str,
+        ns: str = "tf2_buffer_server",
```

Comment:
I guess this makes sense as it is the default name of the buffer server however, if this does not go through in the PR upstream for some reason, we would need to add it again to all our buffer clients. What do you think, we take the risk or not? It's nice not to have to write it everywhere, it can be confusing where it comes from.

Also, makes me think, will we need multiple buffer servers at some point :thinking: